clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/remote/RemoteCacheContainerResourceDefinition.java

```diff
@@ -147,6 +156,12 @@ public static void buildTransformation(ModelVersion version, ResourceTransformat
         } else {
             ResourceTransformationDescriptionBuilder builder = parent.addChildResource(RemoteCacheContainerResourceDefinition.WILDCARD_PATH);
 
+            if (InfinispanModel.VERSION_13_0_0.requiresTransformation(version)) {
+                builder.getAttributeBuilder()
+                        .setDiscard(new DiscardAttributeValueChecker(false, true, ModelNode.FALSE), Attribute.PROPERTIES.getDefinition())
```

Comment:
Given that this is a map attribute, when is this value ever FALSE?
Why not use DiscarcAttributeChecker.UNDEFINED?

Reply:
Thanks for spotting this, fixed + fixed the corresponding rejection test.